packages/editor/src/store/selectors.js

```diff
@@ -1796,3 +1796,7 @@ export const getPostTypeLabel = createRegistrySelector(
 export function isPublishSidebarOpened( state ) {
 	return state.publishSidebarActive;
 }
+
+export const __experimentalIsAdminUser = createRegistrySelector(
```

Comment:
I mean, more than isAdmin, it would be `canDeleteSettings`.

I think only admins can delete settings, but we have to take into account the difference between an admin and a superadmin in multisites. And make sure it works as expected in both.

Reply:
We will explore the possibility of using [canUserEditEntityRecord](https://github.com/WordPress/gutenberg/blob/9659455eed4343c455760b2f9076fe3a5bdbf2d7/packages/core-data/src/selectors.ts#L1164).